edgecast/resources/cps/data_source_cert_order_statuses.go

```diff
@@ -0,0 +1,84 @@
+// Copyright 2022 Edgecast Inc., Licensed under the terms of the Apache 2.0
+// license. See LICENSE file in project root for terms.
+
+package cps
+
+import (
+	"context"
+	"log"
+
+	"terraform-provider-edgecast/edgecast/api"
+
+	"github.com/EdgeCast/ec-sdk-go/edgecast/cps/appendix"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/diag"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+	"github.com/kr/pretty"
+)
+
+func DataSourceCertOrderStatuses() *schema.Resource {
+	return &schema.Resource{
+		ReadContext: DataSourceCertOrderStatusesRead,
+
+		Schema: map[string]*schema.Schema{
+			"id": {
+				Type:        schema.TypeString,
+				Computed:    true,
+				Description: "Indicates the relative path to an endpoint through which you may retrieve a list of certificate order statuses.",
+			},
+			"type": {
+				Type:        schema.TypeString,
+				Computed:    true,
+				Description: "Returns 'Collection'.",
+			},
+			"items": {
```

Comment:
Perhaps we should use the singular noun for the collection here? Since it's a set it would be `cert_order_status". Thoughts?